openfasoc/generators/glayout/glayout/flow/blocks/Divide_by_two_SystemsGenesys/latch_design.py

```diff
@@ -0,0 +1,49 @@
+from gdsfactory import Component
+from glayout.flow.pdk.sky130_mapped import sky130_mapped_pdk as sky130
+from glayout.flow.primitives.fet import nmos
+from glayout.flow.routing.straight_route import straight_route
+from glayout.flow.routing.c_route import c_route
+
+def create_nmos_latch_layout(pdk):
```

Comment:
It would be good to generate the spice netlist for this. Refer to https://github.com/idea-fasoc/OpenFASOC/blob/e7f461123dd9e95216ac9cd59f057c8922fe47de/openfasoc/generators/glayout/glayout/flow/blocks/composite/opamp/opamp.py#L26